test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibrary.cs

```diff
@@ -577,5 +577,15 @@ public void It_passes_ridless_target_to_compiler()
                 .Should()
                 .Pass();
         }
+
+        [Fact]
+        public void It_restores_only_ridless_tfm()
```

Comment:
Full msbuild means the one in VS. Pasting my comment to this thread as well:

I assume it's not failing because the Jenkins machines won't have the corresponding change in NuGet.targets that adds a condition to skip import with these properties. When we move Jenkins to a build with the VS side change then it'll start failing